tests/test_meshtastic_utils_client_cleanup_coverage.py

```
@@ -0,0 +1,338 @@
+            ),
+            patch("mmrelay.meshtastic_utils._disconnect_ble_by_address"),
+            patch(
+                "mmrelay.meshtastic_utils._validate_ble_connection_address",
+                return_value=False,
+            ),
+            patch("mmrelay.meshtastic_utils._disconnect_ble_interface"),
+            patch("mmrelay.meshtastic_utils.logger"),
+        ):
+            saved_iface = None
+
+            def _capture_iface(self: Any, **kwargs: object) -> None:
```

Ruff (0.15.9)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `self`

(ANN401)